cwms-data-api/src/main/java/cwms/cda/data/dao/JooqDao.java

```diff
@@ -69,6 +68,8 @@
 public abstract class JooqDao<T> extends Dao<T> {
     protected static final int ORACLE_CURSOR_TYPE = -10;
     private static final FluentLogger logger = FluentLogger.forEnclosingClass();
+    public static final int FETCH_SIZE = 1000;
```

Comment:
I feel like this should be `DEFAULT_FETCH_SIZE` and `DEFAULT_SMALL_FETCH_SIZE` or the like.

Over time we will likely find that we need to fine tune this for different queries though this is a solid baseline, especially over the apparent default of 10.

Reply:
If wanting to tune this, I recommend that this variable be changed to public static getDefaultFetchSize method that supports a System property override if System property changes are easily done.

Reply:
I doubt we'd want to do a global override of the fetch size. If it's tweaked, it's going to be per query based on that queries specific need.